scripts/core/schemas/user.py

```diff
@@ -0,0 +1,29 @@
+import re
+from typing import Optional
+
+from pydantic import Field, field_validator
+
+from . import BaseSchema
+
+
+class User(BaseSchema):
+    first_name: str
+    middle_name: Optional[str] = None
+    last_name: str
+    username: str
+    email: str

```

Comment:
There is an email & password validator inbuilt available with pydantic which can be used directly.